pages/in/matters/redirect.vue

```diff
@@ -0,0 +1,7 @@
+<script>
+export default {
+  fetch({ redirect, query }) {
+    redirect(302, { name: 'in-getapp', query: { from: 'matters', ...query } });
```

Comment:
I think `from` is already reserved for LikeCoin button along with `referrer`? To prevent confusion, perhaps rename it to `source`?